src/test/java/uk/gov/ons/ctp/common/event/EventPublisherTest.java

```diff
@@ -235,4 +237,34 @@ public void sendEventAddressModificationPayload() throws Exception {
         newAddress.getUprn(),
         event.getPayload().getAddressModification().getNewAddress().getUprn());
   }
+
+  /** Test event message with FeedbackResponse payload */
+  @Test
+  public void sendEventFeedbackPayload() throws Exception {
+
+    Feedback feedbackResponse = new Feedback();
+    feedbackResponse.setPageUrl("url-x");
+    feedbackResponse.setPageTitle("randomPage");
+    feedbackResponse.setFeedbackText("Bla bla bla");
+
+    ArgumentCaptor<FeedbackEvent> eventCapture = ArgumentCaptor.forClass(FeedbackEvent.class);
+
+    String transactionId =
+        eventPublisher.sendEvent(
+            EventType.FEEDBACK, Source.RESPONDENT_HOME, Channel.RM, feedbackResponse);
```

Comment:
Channel should be other than RM - the source is correct (Which service elicited the event), but channel should reflect that it came from a respondent thru respondent home - whichever is appropriate - don't have the choices in front of me.
Channel is used to indicate HOW it came into the source ie Assisted Digital, ordinary respondent, etc